### PR TITLE
[AFID-306] Update UI for "Underpaid tax and debts"

### PR DESCRIPTION
**Pull request by Adam-Shahzad**
removed message
removed column,
reformatted dates

### CODE REVIEW

File: conf/messages [hunk 1]
```diff
@@ -130,15 +126,14 @@ employmenthistory.select.tax.year.error.message = Select a tax year
 
 ################################## Tax Account ####################################################
 employmenthistory.tax-account.header=Underpaid tax and other debts
-employmenthistory.tax-account.caveat.text=You might need this data if you fill in a Self-Assessment return for your client, SA110 boxes 7, 8 and 9.
 #CY
-employmenthistory.tax-account.potential-underpayment.title=Underpaid tax for {0} included in your clients tax code for {1}
+employmenthistory.tax-account.potential-underpayment.title=Underpaid tax for {0} to {1} included in your clients tax code for {2} to {3}
```
Comment (Smudgey) [inline conf/messages]: Missing an apostrophe in the word clients

File: conf/messages [hunk 2]
```diff
@@ -130,15 +126,14 @@ employmenthistory.select.tax.year.error.message = Select a tax year
...
 employmenthistory.tax-account.potential-underpayment.summary = Why does this differ from the amount on the P2 Coding Notice?
 employmenthistory.tax-account.potential-underpayment.content = This data reflects the up to date information for your client.
 #CY-1
-employmenthistory.tax-account.underpayment-amount.title=Underpaid tax for earlier years included in your clients tax code for {0}
+employmenthistory.tax-account.underpayment-amount.title=Underpaid tax for earlier years included in your clients tax code for {0} to {1}
 #CY-1
-employmenthistory.tax-account.outstanding.debt.title=Outstanding debt included in your clients tax code for {0}
+employmenthistory.tax-account.outstanding.debt.title=Outstanding debt included in your clients tax code for {0} to {1}
```
Comment (Smudgey) [inline conf/messages]: Apostrophe here too

File: conf/messages [hunk 3]
```diff
@@ -130,15 +126,14 @@ employmenthistory.select.tax.year.error.message = Select a tax year
...
 employmenthistory.tax-account.header=Underpaid tax and other debts
-employmenthistory.tax-account.caveat.text=You might need this data if you fill in a Self-Assessment return for your client, SA110 boxes 7, 8 and 9.
 #CY
-employmenthistory.tax-account.potential-underpayment.title=Underpaid tax for {0} included in your clients tax code for {1}
+employmenthistory.tax-account.potential-underpayment.title=Underpaid tax for {0} to {1} included in your clients tax code for {2} to {3}
 employmenthistory.tax-account.potential-underpayment.summary = Why does this differ from the amount on the P2 Coding Notice?
 employmenthistory.tax-account.potential-underpayment.content = This data reflects the up to date information for your client.
 #CY-1
-employmenthistory.tax-account.underpayment-amount.title=Underpaid tax for earlier years included in your clients tax code for {0}
+employmenthistory.tax-account.underpayment-amount.title=Underpaid tax for earlier years included in your clients tax code for {0} to {1}
```
Comment (Smudgey) [inline conf/messages]: Apostrophe again

Comment (Smudgey) [reply conf/messages]: This text is wrong, it's missing 'tax year' towards the end. Should read:

Underpaid tax for earlier years included in your client's tax code for tax year 2016 to 2017

File: conf/messages (hunk 1 above)
Comment (Smudgey) [reply conf/messages]: This is missing 'tax year' as well.

Should read:

Underpaid tax for tax year 2016 to 2017 included in your client's tax code 2017 to 2018

File: conf/messages (hunk 2 above)
Comment (Smudgey) [reply conf/messages]: Missing 'tax year' here too

Outstanding debt included in your client's tax code for tax year 2016 to 2017

**Comment by Adam-Shahzad** [comment]
Made changes you requested Plus some more we missed such as removing the word other from the title.

**Pull request merged**